framework/src/org/checkerframework/framework/util/element/SuperTypeApplier.java

```diff
@@ -81,7 +81,16 @@ public int getElementIndex() {
      */
     @Override
     public int getTypeCompoundIndex(Attribute.TypeCompound anno) {
-        return anno.getPosition().type_index;
+        int type_index = anno.getPosition().type_index;
+        // TODO: this is a temporary workaround of a bug in jsr308-langtools
```

Comment:
Is this a jsr308-langtools bug or a general langtools bug?  Has a bug report been summited?